kpi/models/import_export_task.py

```diff
@@ -477,27 +488,26 @@ def export_upload_to(self, filename):
     return posixpath.join(self.user.username, 'exports', filename)
 
 
-class ProjectViewExportTask(ImportExportTask):
-    uid = KpiUidField(uid_prefix='pve')
-    result = PrivateFileField(upload_to=export_upload_to, max_length=380)
+class ExportTaskMixin:
+
+    @property
+    def default_email_subject(self) -> str:
+        return 'Report Complete'
+
+    def _get_export_details(self) -> tuple:
+        return self.data['type'], self.data['view']
```

Comment:
This should account for `data` not having a `view` key since we don't need one for access logs exports. the filename can just be `export_type-username-time` for tasks without a view.